packages/manager/modules/billing/src/autoRenew/autorenew.html

```diff
@@ -1,4 +1,7 @@
-<oui-header data-heading="{{:: 'billing_title' | translate}}">
+<oui-header
+    data-heading="{{:: 'billing_title' | translate}}"
+    data-description="{{:: $ctrl.coreConfig.getRegion() !== 'US' ? 'billing_description' : '' | translate}}"
```

Comment:
Question: Empty heading for US? is it as per the requirements?

Reply:
It is planned in the future to have US content, but it is not yet adapted and ready.